### ISSUE TITLE
Support for itemprop 

### ISSUE THREAD

**Issue by graham73may** [ronco/ember-cli-meta-tags#26]
One of those projects where all the sharers are required so I'm adding in 'The Full Monty' from:
https://moz.com/blog/meta-data-templates-123

This includes:
```
<!-- Schema.org markup for Google+ -->
<meta itemprop="name" content="The Name or Title Here">
<meta itemprop="description" content="This is the page description">
<meta itemprop="image" content="http://www.example.com/image.jpg">
```

**Comment by ronco** [comment]
"Full Monty" nice!

Looks reasonable. Should be a simple add. Would you like to submit a PR? If
not I can get to this in the next couple of days.

Thanks!

**Comment by graham73may** [comment]
Pull request submitted - #27

**Comment by ronco** [comment]
@graham73may just published 1.0.1 to npm with this.  Thanks for the PR!
